documentation/Configs.md

```diff
@@ -84,3 +84,12 @@ for the reset API and how to use it.
 **Purpose**: This config fetches default settings from a remote URL. The default URL is set to
 `https://raw.githubusercontent.com/GPII/universal/master/testData/defaultSettings/defaultSettings.win32.json5`
 in this config. See [Reset Computer](ResetComputer.md) for what are default settings.
+
+### Handle Revision Endpoint in Production
+
+**Config file**: [`%universal/gpii/configs/gpii.config.production.local.flowManager.revision.json5`](../gpii/configs/gpii.config.production.local.flowManager.revision.json5)
```

Comment:
A couple of thoughts:
1. This config should be an add-on config that will be up to config creators to add when they need to. We probably don't want to create a full untrusted config.
2. Based on 1, this config should be in `gpii/node_modules/flowManager/configs` with a name `gpii.flowManager.config.untrusted.remoteSolutionRegistry.base.json5`. It will decide more than whether to fetch revision hash from the Cloud, which is just one step of the remote SR fetching. It determines whether the solutions registry will be fetched from the Cloud.

Reply:
Done.